presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -311,16 +307,14 @@ private JdbcOutputTableHandle beginWriteTable(ConnectorTableMetadata tableMetada
                 }
                 columnNames.add(columnName);
                 columnTypes.add(column.getType());
-                columnList.add(new StringBuilder()
-                        .append(quoted(columnName))
-                        .append(" ")
-                        .append(toSqlType(column.getType()))
-                        .toString());
+                columnList.add(format("%s %s", quoted(columnName), toSqlType(column.getType())));
             }
-            Joiner.on(", ").appendTo(sql, columnList.build());
-            sql.append(")");
 
-            execute(connection, sql.toString());
+            String sql = format(
+                    "CREATE TABLE %s (%s)",
+                    quoted(catalog, schema, temporaryName),
+                    String.join(", ", columnList.build()));
```

Comment:
static import?

String.join?  o_O I thought that such magic is only available in python.